dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresVectorStoreOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.Postgres;
+
+/// <summary>
+/// Options when creating a <see cref="PostgresVectorStore"/>.
+/// </summary>
+public sealed class PostgresVectorStoreOptions
+{
+    /// <summary>
+    /// Gets or sets the default vector size to use when creating a new vector.
+    /// </summary>
+    public int DefaultVectorSize { get; init; } = 100;
```

Comment:
It might be better to throw if a user didn't provide the dimensions for a vector.  The number of a dimensions vary by the embedding model used, and has to match the embedding model.  Any default that we provide would be unlikely to match that and might make it hard for the user to realise they've made a mistake by not specifying it.